src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -120,6 +120,9 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
```

Comment:
@SteveL-MSFT, your last commit had 6 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

PSPath to native commands.PSPath with ~/path works
```powershell
Expected 0, but got 1.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\NativeExecution\NativeCommandArguments.Tests.ps1: line 107
107:         $LASTEXITCODE | Should -Be 0
```
PSPath to native commands.PSPath with ~ works
```powershell
Expected 0, but got 1.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\NativeExecution\NativeCommandArguments.Tests.ps1: line 113
113:         $LASTEXITCODE | Should -Be 0
```
Test suite for validating automounted PowerShell drives.Validating automounting FileSystem drives.Test automounting using subst.exe
```powershell
Expected exactly 'Drive found', but got @('Drive found', 'Invalid parameter - Y:\').
at <ScriptBlock>, D:\a\1\s\test\powershell\Provider\Pester.AutomountedDrives.Tests.ps1: line 40
40:            & $powershell -noprofile -command "& '$AutomountSubstDriveScriptPath' -FullPath '$substDir'" | Should -BeExactly "Drive found"
```
Test suite for validating automounted PowerShell drives.Validating automounting FileSystem drives from modules.Test automounting using subst.exe
```powershell
Expected exactly 'Drive found', but got @('Drive found', 'Invalid parameter - X:\').
at <ScriptBlock>, D:\a\1\s\test\powershell\Provider\Pester.AutomountedDrives.Tests.ps1: line 51
51:            & $powershell -noprofile -command "& '$AutomountSubstDriveScriptPath' -useModule -FullPath '$substDir'" | Should -BeExactly "Drive found"
```
PSPath to native commands.PSPath with ~/path works
```powershell
Expected 0, but got 1.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\NativeExecution\NativeCommandArguments.Tests.ps1: line 107
107:         $LASTEXITCODE | Should -Be 0
```